packages/flutter/test/material/tabs_utils.dart

```diff
@@ -0,0 +1,239 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+RenderParagraph getTabText(WidgetTester tester, String text) {
+  return tester.renderObject<RenderParagraph>(find.text(text));
+}
```

Comment:
Should this be a `find.descendant` with a widget predicate that makes sure that this RenderParagraph is coming from within the TabBar?

Example: https://github.com/flutter/flutter/blob/15f8ef71d920dff34ae1b187d7cd5bf5a05c104a/dev/integration_tests/flutter_gallery/test/drawer_test.dart#L79